ynr/apps/moderation_queue/views.py

```diff
@@ -350,11 +342,10 @@ def crop_and_upload_image_to_popit(
             },
         )
 
-        if make_primary:
-            sorl_delete(person.primary_image.file, delete_file=False)
-            # Update the last modified date, so this is picked up
-            # as a recent edit by API consumers
-            person.save()
+        sorl_delete(person.person_image.file, delete_file=False)
```

Comment:
We are not clear if this is still related - left in for now as we felt that changing a primary image is the action as changing an image now

Reply:
Yep, I don't remember the details but it's from this commit: https://github.com/DemocracyClub/yournextrepresentative/commit/52f54eed8e15eebc39516c1f52b7c58b4c33b08e

I think the problem was that the thumbnail of old images was hanging about after changing the person image. `sorl_delete` removes the thumbnail from the cache, and forces it to be remade on the next request.